spec.emu

```diff
@@ -13,22 +13,73 @@ contributors:
   - Leo Balter
   - Caridy Patiño
   - Rick Waldron
+  - Ashley Claymore
 </pre>
 
-<emu-clause id="sec-isobjectorsymbol" aoid="IsObjectOrSymbol">
-  <h1>IsObjectOrSymbol ( _argument_ )</h1>
-  <p>The abstract operation IsObjectOrSymbol takes argument _argument_ (an ECMAScript language value) and determines if its Type is Object or Symbol. This operation performs the following steps when called:</p>
+<emu-clause id="sec-ecmascript-data-types-and-values" aoid="Type">
+  <h1>ECMAScript Data Types and Values</h1>
+  <emu-clause id="sec-object-type">
+    <h1>The Object Type</h1>
+    <emu-clause id="sec-well-known-intrinsic-objects">
+      <h1>Well-Known Intrinsic Objects</h1>
+      <emu-table id="table-well-known-intrinsic-objects" caption="Well-Known Intrinsic Objects" oldids="table-7">
+        <table>
+          <tr>
+            <th>
+              Intrinsic Name
+            </th>
+            <th>
+              Global Name
+            </th>
+            <th>
+              ECMAScript Language Association
+            </th>
+          </tr>
+          <tr>
+            <td>
+              <ins>%CanBeHeldWeakly%</ins>
```

Comment:
We create this as a new intrinsic so it can be shared across `WeakMap.isValidKey`, `WeakSet.isValidValue`, `WeakRef.isValidTarget` and `FinalizationRegistry.isValidTarget`.